examples/jsm/nodes/core/VarNode.js

```diff
@@ -52,6 +52,7 @@ export default VarNode;
 
 export const temp = nodeProxy( VarNode );
 
-addNodeElement( 'temp', temp );
+addNodeElement( 'temp', temp ); // @TODO: Will be removed in the future
+addNodeElement( 'toVar', ( ...params ) => temp( ...params ).append() );
```

Comment:
Maybe we can rename this back to `temp`? It seems like better name and the behavior is almost unchanged (except that now it forces the variable to be generated immediately in the shader flow, if I understand correctly), so it will not be a so breaking change.

Reply:
I think that the conversion variables used in the "chain" could have the prefix "to". It seems to sound more like the conversions we see in any other API, for example `value.toVec3()` instead of `value.vec3()` the same I see for `value.toVar()`

Reply:
I don't really see "temping" a node as a conversion... I agree about `toVec3`, but for `temp` I still think this is the best name.

Reply:
Maybe `createVar()`? 

I don't think `temp` is a bad name, it just seems like it wasn't clear that it declares a variable, I think doubts like [this ](https://github.com/mrdoob/three.js/pull/26211#discussion_r1222590624) could be avoided.

Reply:
I think that the name of this element should be short, because it is one of the most used ones. I also don't really think name `toVar` makes it significantly more understandable for someone who doesn't know what this element do...
In my opiniion `temp` is best both for being short and being understandable ("temping" a node creates a temp variable with it).